### ISSUE TITLE
Create a waiting list signup page for mozilla.social

### ISSUE THREAD

**Issue by pmac** [mozilla/bedrock#13635]
## Description

The Mozilla Social team wants to move their waiting list from the current system to use our standard bedrock -> basket -> CTMS -> Acoustic one. We will need to create the form on bedrock and host it at a URL they've listed in the doc linked below (or one we feel is better if we want to pitch them something else).

[Spec doc](https://docs.google.com/document/d/1lczzwGgkNi50IiH3hlaR6HjNYnzw2PKHo3tr2Tdicdk/edit#heading=h.lxk671tolfxh)

The [existing form is here](https://prod.oidc-proxy.prod.webservices.mozgcp.net/waitlist).

The extra non-standard fields will pass through basket to CTMS as long as they're named appropriately with the waiting list's prefix, much like the country and platform fields on other waiting list forms. 

For example:

The slug for this waiting list is `mozilla-social-waitlist`, so the extra fields should be something like:

* `mozilla_social_twitter_handle`
* `mozilla_social_mastodon_handle`

In other words, you remove `-waitlist` from the name, replace the `-` with `_`, and add a unique field name after.
